tools/bcdb_mock/pkg/directory/farms_handlers.go

```diff
@@ -90,3 +91,36 @@ func (s *FarmAPI) getFarm(r *http.Request) (interface{}, mw.Response) {
 
 	return farm, nil
 }
+
+func (s *FarmAPI) delete(r *http.Request) (interface{}, mw.Response) {
```

Comment:
Why do we need that?
I mean in block chain database, the "delete" should not be allowed (or at least the data should not get deleted from the database, may be only marked as deleted)

Reply:
You are right I should not delete anything but use another mechanism to marked them as deleted. I'll remove from this PR and open an issue to discuss how to handle clean up in a better way.